scripts/run_backend_tests.py

```diff
@@ -603,7 +603,39 @@ def main(args: Optional[List[str]] = None) -> None:
     if parsed_args.generate_coverage_report:
         subprocess.check_call([sys.executable, '-m', 'coverage', 'combine'])
         report_stdout, coverage = check_coverage(True)
+        print('')
+        print(
+            '+----------------------------------------------------------------+'
+            )
+        print(
+            '|-------- '
+            'SUMMARY OF THE FILES WITH INCOMPLETE COVERAGE '
+            '---------|'
+            )
+        print(
+            '+----------------------------------------------------------------+'
+            )
+        print('')
         print(report_stdout)
+        if coverage != 100:
+            print(
+                'WARNING: Backend test coverage is below 100%.\n'
+                'The rightmost "Missing" column above'
+                ' shows lines that are still uncovered.'
+            )
+
+            print(
+                'Please add tests for scenarios that'
+                ' exercise those lines of code\n'
+                'so that there are no uncovered lines in each file.'
+            )
+
+            print(
+                'For more information, please see our\n'
+                '[backend tests wiki page]\n'
```

Comment:
Why do you need an \n at the end of this?

Reply:
@seanlip I included ' \n' here, following @U8NWXD's recommendation to ensure that no lines in the code or output exceed 80 characters.

Reply:
Then this doesn't seem to be reflected in your proof of changes? Please keep the proof of changes in your PR description up to date.

Reply:
@seanlip i have updated the proof of changes. Thanks.